python/python/bystro/api/cli.py

```diff
@@ -572,6 +640,14 @@ def main():
     )
     jobs_parser.set_defaults(func=get_jobs)
 
+    query_parser = subparsers.add_parser("query", help="The OpenSearch query string query, e.g. (cadd: >= 20)")
+    query_parser.add_argument("--dir", default=DEFAULT_DIR, help="Where Bystro API login state is saved")
+    query_parser.add_argument("--query", required=True, help="The OpenSearch query string query, e.g. (cadd: >= 20)")
+    query_parser.add_argument("--size", default=10, type=int, help="How many records (default: 10)")
+    query_parser.add_argument("--from_", default=0, type=int, help="The first record to return from the matching results. Used for pagination.")
```

Comment:
minor nit: let's remove the period after "Used for pagination" to match the periodless other help strings